Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -904,13 +921,10 @@ class CombinatorialKalmanFilter {
           tipState.nMeasurements++;
         }
 
-        // Check if need to stop this branch
-        if (!m_extensions.branchStopper(tipState, trackState)) {
```

Comment:
I don't understand why the branch stopper was previously called again here, so I guess this is an improvement. Can you just confirm that the change was intended?

Reply:
That is intentional yes.  I think this was used to potentially abort after a measurement was selected but I don't think this should be done. We should only call the branch aborter once after filtering. The measurement selector can already cut with all the information at hand.